src/app/home/home.component.html

```diff
@@ -20,8 +20,7 @@
             <div class="left-side text-space">
                 <div class="title">Getting Started Today</div>
                 <p>
-                    DevExtreme Subscription includes Optimized HTML5 JavaScript SPA
-                    Framework and 30+ High Performance HTML5 JavaScript Widgets.
+                    DevExtreme Subscription includes 65+ High Performance HTML5 JavaScript Widgets.
```

Comment:
Do all the words have to be in upper case?

Reply:
> 
> 
> Do all the words have to be in upper case?

it was written in the same way https://devexpress.github.io/golfclub/home